pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -3374,6 +3376,40 @@ public TableStats getTableStats(String tableNameWithType) {
     return new TableStats(creationTime);
   }
 
+  /**
+   * Returns map of tableName to list of live brokers
+   * @return Map of tableName to list of ONLINE brokers serving the table
+   */
+  public Map<String, List<InstanceInfo>> getLiveBrokers() {
```

Comment:
We should rename this method to reflect that it returns table to hosts mapping and not tenant to hosts mapping.

Maybe something like - `getLiveBrokersForTables`

Reply:
Renamed to `getTableToLiveBrokersMapping`. `getLiveBrokersForTables` already exists that takes tableName as param